plugins/automap/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@flatfile/plugin-automap",
+  "version": "0.0.1",
+  "description": "A plugin for auto mapping imported files when confidence levels are high.",
+  "registryMetadata": {
+    "status": "Coming soon",
+    "category": "schema"
+  },
+  "engines": {
+    "node": ">= 12"
```

Comment:
❔ Other plugins use this. Is that accurate? I was thinking more like ">= 16" from personal experience

Reply:
16 yea